### ISSUE TITLE
W1D4 (Sat Jan 25): Backpropagation

### ISSUE THREAD

**Issue by hieple7985** [hieple7985/personal.me#65]
## Learning (2h)
- [ ] 3B1B Ch 6: Inverse matrices
- [ ] 3B1B Ch 7: Dot products deep dive

## Project (2h)
- [ ] Backward pass in Layer
  - `backward(self, dL_dout)` → returns dL_din
  - Store gradients for W, b
- [ ] Backward pass in Network
  - Chain gradients backward through layers
- [ ] `update(self, learning_rate)`

## Check-in
**Done:** 
**Learned:**
**Blocked:**
**Next:** W1D5 - Training + Test

---

*Due: Jan 25 | Week 1: [#49](https://github.com/hieple7985/personal.me/issues/49) | Master: [#48](https://github.com/hieple7985/personal.me/issues/48)*

**Comment by hieple7985** [comment]
Plan period ended (Jan-Mar 2026). Closing stale planning issue.
